{
	"id": "CVE-2024-22341",
	"sourceIdentifier": "psirt@us.ibm.com",
	"vulnStatus": "Awaiting Analysis",
	"published": "2025-02-22T01:15:10.507",
	"lastModified": "2025-02-22T01:15:10.507",
	"descriptions": [
		{
			"lang": "en",
			"value": "IBM Watson Query on Cloud Pak for Data 4.0.0 through 4.0.9, 4.5.0 through 4.5.3, 4.6.0 through 4.6.6, 4.7.0 through 4.7.4, and 4.8.0 through 4.8.7 could allow unauthorized data access from a remote data source object due to improper privilege management."
		},
		{
			"lang": "es",
			"value": "IBM Watson Query en Cloud Pak for Data 4.0.0 a 4.0.9, 4.5.0 a 4.5.3, 4.6.0 a 4.6.6, 4.7.0 a 4.7.4 y 4.8.0 a 4.8.7 podría permitir el acceso no autorizado a datos desde un objeto de origen de datos remoto debido a una gestión de privilegios inadecuada."
		}
	],
	"references": [
		{
			"source": "psirt@us.ibm.com",
			"url": "https://www.ibm.com/support/pages/node/7183851"
		}
	],
	"metrics": {
		"cvssMetricV31": [
			{
				"source": "psirt@us.ibm.com",
				"type": "Primary",
				"cvssData": {
					"version": "3.1",
					"vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:H/I:N/A:N",
					"attackVector": "NETWORK",
					"attackComplexity": "HIGH",
					"privilegesRequired": "LOW",
					"userInteraction": "NONE",
					"scope": "UNCHANGED",
					"confidentialityImpact": "HIGH",
					"integrityImpact": "NONE",
					"availabilityImpact": "NONE",
					"baseScore": 5.3,
					"baseSeverity": "MEDIUM"
				},
				"exploitabilityScore": 1.6,
				"impactScore": 3.6
			}
		]
	},
	"weaknesses": [
		{
			"source": "psirt@us.ibm.com",
			"type": "Primary",
			"description": [
				{
					"lang": "en",
					"value": "CWE-269"
				}
			]
		}
	]
}
